### PR TITLE
chore(storybook): Fix typo for prev button of calendar story

### LINKED ISSUE #159: [intuitiveness] ✏️ Fix Calendar.tsx text to be easier to understand
## Description
two buttons for navigating prev and next calendar are using the same character (>)
I wanna suggest a different character, '<' for 'button for navigating prev'
I think it's more 

## Current
src/stories/Calendar.tsx

```jsx
<button aria-label="button for navigating to prev calendar" onClick={navigation.toPrev}>{'>'}</button>
<button aria-label="button for navigating to today calendar" onClick={navigation.setToday}>TODAY</button>
<button aria-label="button for navigating to next calendar" onClick={navigation.toNext}>{'>'}</button>
```

## Expected
```jsx
<button aria-label="button for navigating to prev calendar" onClick={navigation.toPrev}>{'<'}</button>
<button aria-label="button for navigating to today calendar" onClick={navigation.setToday}>TODAY</button>
<button aria-label="button for navigating to next calendar" onClick={navigation.toNext}>{'>'}</button>
```

## Current References
image

### PR DESCRIPTION
**Pull request by manudeli**
fix text for improving intuitiveness of prev button

closes #159

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
     <td align="center"><a href="https://velog.io/@bigsaigon333"><img src="https://avatars.githubusercontent.com/u/31029000?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Danny Kim</b></sub></a><br /><a href="#maintenance-bigsaigon333" title="Maintenance">🚧</a></td>
     <td align="center"><a href="http://gladys.codes"><img src="https://avatars.githubusercontent.com/u/40550185?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Gladys Álvarez</b></sub></a><br /><a href="https://github.com/veccu/react-calendar/commits?author=gladyscodes" title="Documentation">📖</a></td>
     <td align="center"><a href="https://techoi.github.io"><img src="https://avatars.githubusercontent.com/u/47510626?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Jeemin Choi</b></sub></a><br /><a href="https://github.com/veccu/react-calendar/issues?q=author%3Atechoi" title="Bug reports">🐛</a></td>
+    <td align="center"><a href="https://github.com/manudeli"><img src="https://avatars.githubusercontent.com/u/61593290?v=4=100" width="100px;" alt=""/><br /><sub><b>Jonghyeon Ko</b></sub></a><br /><a href="https://github.com/h6s-dev/h6s/issues?q=author%3Amanudeli" title="Content">🖋</a></td>
```
Comment (JaeYeopHan) [inline README.md:26]: remove this line please

**Review by JaeYeopHan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (manudeli) [reply README.md]: @JaeYeopHan oh sorry, I misunderstood it that I needed to add my name in this table!

I removed this line now!

**Comment by JaeYeopHan** [comment]
@all-contributors please add @manudeli for docs

File: README.md (hunk 1 above)
Comment (JaeYeopHan) [reply README.md]: thanks :)

**Comment by JaeYeopHan** [comment]
@all-contributors please add @manudeli for docs

**Pull request merged**